.devcontainer/post-create.sh

```diff
@@ -2,26 +2,17 @@
 
 set -e # Fail the whole script on first error
 
+gem install bundler -v $(tail -n 1 Gemfile.lock)
+
 # Fetch Ruby gem dependencies
 bundle config path 'vendor/bundle'
 bundle config with 'development test'
-bundle install
-
-# Make Gemfile.lock pristine again
-git checkout -- Gemfile.lock
+bundle check || bundle install
 
 # Fetch Javascript dependencies
 corepack prepare
 yarn install --immutable
 
-# [re]create, migrate, and seed the test database
-RAILS_ENV=test ./bin/rails db:setup
-
 # [re]create, migrate, and seed the development database
-RAILS_ENV=development ./bin/rails db:setup
-
-# Precompile assets for development
-RAILS_ENV=development ./bin/rails assets:precompile
+./bin/rails db:setup
```

Comment:
We have this (atypical for rails apps) hard check for the presence of RAILS_ENV (See boot.rb), and I think this command will fail as-is?

My personal preference would be to remove that check, but apparently it has helped stop bad things from happening to good people.

Reply:
So, in the "docker-compose.yml" we already have the RAILS_ENV defined there.